frontend/app/forms/coversheet/coversheet.web.component.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Scoped } from "kremling";
+import TextInput from "../inputs/text-input.component.js";
+import Select from "../inputs/select.component.js";
+import TextArea from "../inputs/text-area.component.js";
+import FormThatPrints from "../inputs/form-that-prints.component.js";
+import Section from "../inputs/section.component.js";
+
+export default class Coversheet_Web extends React.Component {
+  render() {
+    return (
+      <Scoped css={css}>
+        <FormThatPrints>
```

Comment:
Each form should have at least one `<Section>` I think. Maybe add one here called "1. Personal Information"

See https://github.com/JustUtahCoders/utahexpungements.org/blob/master/frontend/app/forms/application-for-coe/application-for-coe.web.component.js#L27 for example

Reply:
👍